websites/J/JavaScript/dist/metadata.json

```diff
@@ -6,13 +6,12 @@
     },
     "url": "developer.mozilla.org",
     "description": {
-      "es": "Pone en tu presencia lo que estas viendo en los docs JS",
       "en": "Show the things that are you seeing in the JS docs"
     },
-    "service": "JavaScript | MDN",
+    "service": "JavaScript",
     "version": "1.0.0",
-    "logo": "https://vyrekxd.is-inside.me/jNHWfjeY.png",
-    "thumbnail": "https://vyrekxd.is-inside.me/hkWnnOcf.png",
+    "logo": "https://i.imgur.com/cpLEEoE.png",
```

Comment:
upload a bigger version of the logo (at least 512x512)

Reply:
okay